src/components/Agreement/Agreement.js

```diff
@@ -40,18 +42,18 @@ function AgreementLayout({ agreementDetails }) {
   } = agreementDetails
 
   return (
-    <Split
+    <LayoutColumns
       primary={
         <>
-          <Box>
+          <LayoutBox primary>
```

Comment:
I've been prefixing our custom layout components with `Layout`. This makes the distinction clear vs using the same names as exist in the UI kit.

The `primary` prop is what gives the box extra padding for use in the `primary` slot. Previously the box would inherit this implicitly, which caused problems such as those [described here](https://github.com/aragon/network-dashboard/issues/19)